ui/app/styles/core/switch.scss

```diff
@@ -95,7 +95,7 @@
       content: "";
       height: $size-8;
       position: absolute;
-      top: $size-8 / 5;
+      top: 0.147rem;
```

Comment:
I know we have other sizes of the switch, although I'm not sure we use them anywhere. Do we need to adjust the top position amount on those as well?

Reply:
I checked the other cases and they all looked okay. I really don't like doing anything different than the existing css. Let me spend more time on this tmrw and see if I can figure out why the css can't be what it was from bulma_switch.

Reply:
Okay, I was able to fix this back to the original. I found one use case (in this policy modal) where it is still an issue, but I'll make another ticket for it as I think it's a bigger issue. I'm also not sure when we should be using toggle vs switch. Any thoughts? If it's toggle then I can make another ticket to merge to main to remove all switches and replace with toggle....

Reply:
yeah good point, we can probably do follow on to replace all with toggle, but also to replace all of our toggles with [HDS toggle](https://hds-website-hashicorp.vercel.app/components/form/toggle?tab=code) 🙃